org.yaml

```diff
@@ -47,6 +72,6 @@ spec:
   profile:
     # Intentional no displayName for testing
     email: user{{ index + 1 }}@opentlc.com
-    picture: https://avatars.dicebear.com/api/avataaars/breanna-davison@example.com.svg?background=%23fff
-  memberOf: [user{{ index + 1 }}]
+    picture: https://api.dicebear.com/9.x/thumbs/svg?seed={{ user_images[index % user_images | length] }}
+  memberOf: [user{{ index + 1 }}, developers]
```

Comment:
Question: should we remove the `userN` group entirely? Was there a reason it was added? Seems a little confusing to have a user and group with the same name 1:1.